src/Uno.UI.Composition/Composition/InteractionTracker/IInteractionTrackerOwner.cs

```diff
@@ -0,0 +1,18 @@
+#nullable enable
+
+namespace Microsoft.UI.Composition.Interactions;
+
+public partial interface IInteractionTrackerOwner
+{
+	void CustomAnimationStateEntered(InteractionTracker sender, InteractionTrackerCustomAnimationStateEnteredArgs args);
+
+	void IdleStateEntered(InteractionTracker sender, InteractionTrackerIdleStateEnteredArgs args);
+
+	void InertiaStateEntered(InteractionTracker sender, InteractionTrackerInertiaStateEnteredArgs args);
+
+	void InteractingStateEntered(InteractionTracker sender, InteractionTrackerInteractingStateEnteredArgs args);
```

Comment:
Why not a single method for all states : `EnterState(Interactionracker tracker, InteractionTrackerState state, InteractionTrackerInteractingStateEnteredArgs args)` (we can even remove the `InteractionTrackerState state` if `args` already provide it.

Reply:
This is a public interface